Lib/fontTools/ttLib/tables/otBase.py

```diff
@@ -862,6 +900,13 @@ def getRecordSize(cls, reader):
 		return NotImplemented
 
 	def getConverters(self):
+		try:
+			fmt = self.Format
+		except AttributeError:
+			# some FormatSwitchingBaseTables (e.g. Coverage) no longer have 'Format'
+			# attribute after fully decompiled, only gain one in preWrite before being
+			# recompiled.
+			return []
```

Comment:
I don't find it completely obvious that [] is the right answer in that situation, if it is perhaps explain why in the comment?

Reply:
they have no Format, so they don't/can't have format-specific converters, the only possible thing to return is nothing. These hand-coded otTables.py classes provide a nicer API that breaks away with the converters-driven class model, they are supposed to be at the end of a class tree (as leaves), they are not supposed to contain in turn any low-level converters-driven objects.

Reply:
added comment